coalib/results/result_actions/ShowPatchAction.py

```diff
@@ -79,9 +81,13 @@ def is_applicable(result: Result, original_file_dict, file_diff_dict):
                              file_diff_dict[filename]):
                     nonempty_patches = True
 
-            return nonempty_patches
-        except ConflictError:
-            return False
+            if nonempty_patches:
+                return True
+            return patch_not_generated_msg
```

Comment:
this case is a bit special, not sure how to get that through to the user... but this message isn't it. No idea what message would be better, worst case like "The given patches do not change anything anymore"